src/utils/styles/theme.ts

```diff
@@ -51,6 +51,7 @@ function getGradients(type: 'dark' | 'light') {
         : [colors.lilac, colors.iris],
     ),
     creditPoolText: makeGradient([colors.blueViolet, colors.lavenderBlue]),
+    poolBalanceChart: [makeGradient(['#fc87e2', '#f24cb6']), makeGradient(['#63afdd', '#574cf2'])],
```

Comment:
Use `[] as const`, without this we can read `poolBalanceChart[3]` and not see the ts error